.changeset/gold-clouds-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/table': major
+---
+
+Virtual scrolling, selectable rows, and other major features. Refer to upgrade guide for details
```

Comment:
This should probably be a little bit more descriptive. 

"Rewrite of Table component, which includes support for new features: XYZ. See upgrade guide for more details"